CMake/SlicerMacroBuildQtModule.cmake

```diff
@@ -1,325 +1 @@
-    endif()
-  else()
-    if(NOT DEFINED ${LOADABLEMODULE_NAME}_DEVELOPMENT_INSTALL)
-      set(${LOADABLEMODULE_NAME}_DEVELOPMENT_INSTALL OFF)
-    endif()
-  endif()
-
-  if(NOT LOADABLEMODULE_NO_INSTALL AND ${LOADABLEMODULE_NAME}_DEVELOPMENT_INSTALL)
-    # Install headers
-    file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
-    install(FILES
-      ${headers}
-      ${dynamicHeaders}
-      DESTINATION ${Slicer_INSTALL_QTLOADABLEMODULES_INCLUDE_DIR}/${LOADABLEMODULE_NAME} COMPONENT Development
-      )
-  endif()
-
-  if(BUILD_TESTING AND LOADABLEMODULE_WITH_GENERIC_TESTS)
-
-    set(KIT ${lib_name})
-
-    set(KIT_GENERIC_TEST_SRCS)
-    set(KIT_GENERIC_TEST_NAMES)
-    set(KIT_GENERIC_TEST_NAMES_CXX)
-    SlicerMacroConfigureGenericCxxModuleTests(${LOADABLEMODULE_NAME}
-      KIT_GENERIC_TEST_SRCS
-      KIT_GENERIC_TEST_NAMES
-      KIT_GENERIC_TEST_NAMES_CXX
-      )
-
-    set(CMAKE_TESTDRIVER_BEFORE_TESTMAIN "DEBUG_LEAKS_ENABLE_EXIT_ERROR();" )
-    create_test_sourcelist(Tests ${KIT}CxxTests.cxx
-      ${KIT_GENERIC_TEST_NAMES_CXX}
-      EXTRA_INCLUDE vtkMRMLDebugLeaksMacro.h
-      )
-
-    add_executable(${KIT}GenericCxxTests ${Tests})
-    set_target_properties(${KIT}GenericCxxTests PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${Slicer_BIN_DIR})
-    target_link_libraries(${KIT}GenericCxxTests ${KIT})
-    if(NOT "${LOADABLEMODULE_FOLDER}" STREQUAL "")
-      set_target_properties(${KIT}GenericCxxTests PROPERTIES FOLDER ${LOADABLEMODULE_FOLDER})
-    endif()
-
-    foreach(testname ${KIT_GENERIC_TEST_NAMES})
-      add_test(NAME ${testname}
-               COMMAND ${Slicer_LAUNCH_COMMAND} $<TARGET_FILE:${KIT}GenericCxxTests> ${testname})
-      set_property(TEST ${testname} PROPERTY LABELS ${KIT})
-    endforeach()
-  endif()
-
-endmacro()
-
-
-#
-# Deprecated macro - Will be removed for 4.4
-#
-macro(slicerMacroBuildQtModule)
-  set(options
-    NO_INSTALL
-    NO_TITLE
-    WITH_GENERIC_TESTS
-    )
-  set(oneValueArgs
-    NAME
-    TITLE
-    EXPORT_DIRECTIVE
-    )
-  set(multiValueArgs
-    SRCS
-    MOC_SRCS
-    UI_SRCS
-    INCLUDE_DIRECTORIES
-    TARGET_LIBRARIES
-    RESOURCES
-    )
-  CMAKE_PARSE_ARGUMENTS(QTMODULE
-    "${options}"
-    "${oneValueArgs}"
-    "${multiValueArgs}"
-    ${ARGN}
-    )
-
-  #message(AUTHOR_WARNING
-  #  "Macro 'slicerMacroBuildQtModule' is deprecated, "
-  #  "consider using macro 'slicerMacroBuildLoadableModule'."
-  #  "See http://www.na-mic.org/Bug/view.php?id=2648"
-  #  )
-
-  set(QTMODULE_ARGS)
-  foreach(arg ${oneValueArgs} ${multiValueArgs})
-    list(APPEND QTMODULE_ARGS ${arg} ${QTMODULE_${arg}})
-  endforeach()
-
-  set(QTMODULE_OPTIONS)
-  foreach(option ${options})
-    if(${QTMODULE_${option}})
-      list(APPEND QTMODULE_OPTIONS ${option})
-    endif()
-  endforeach()
-
-  slicerMacroBuildLoadableModule(${QTMODULE_ARGS} ${QTMODULE_OPTIONS})
-endmacro()
+include(${CMAKE_CURRENT_LIST_DIR}/SlicerMacroBuildLoadableModule.cmake)
```

Comment:
I would also think about adding a deprecation message to be displayed whenever this file is included. This would "force" people to include the correct (SlicerMacroBuildLoadableModule) cmake file.
@jcfr: What do you think ?